extras/rya.streams/api/src/main/java/org/apache/rya/streams/api/entity/StreamsQuery.java

```diff
@@ -85,4 +85,19 @@ public boolean equals(final Object o) {
         }
         return false;
     }
+    
+    @Override
+    public String toString() {
+        final StringBuilder sb = new StringBuilder();
+        sb.append("ID: ");
+        sb.append(getQueryId().toString() + "\n");
+        sb.append("Query: ");
+        sb.append(getSparql() + "\n");
```

Comment:
Is "getSparql()" coming from user input? Might want to wrap that in org.apache.rya.api.log.LogUtils.clean(getSparql())

Reply:
Nope.